sanic/app.py

```diff
@@ -514,7 +523,11 @@ def _helper(self, host="127.0.0.1", port=8000, debug=False,
                 protocol=HttpProtocol, backlog=100, stop_event=None,
                 register_sys_signals=True, run_async=False):
         """Helper function used by `run` and `create_server`."""
-
+        if stop_event is not None:
```

Comment:
or we could just not log anything or accept stop_event in `_helper`, since that shouldn't be used directly anyway